cmd/kubeadm/app/cmd/join.go

```diff
@@ -419,6 +421,20 @@ func (j *joinData) Run() error {
 		fmt.Printf("[join] Running pre-flight checks before initializing the new control plane instance\n")
 		preflight.RunInitMasterChecks(utilsexec.New(), initCfg, j.ignorePreflightErrors)
 
+		fmt.Printf("[join] Pulling control-plane images\n")
```

Comment:
should be `fmt.Println` without the `\n` at the end.

the above call is also wrong, but leave it as its out of scope.
`fmt.Printf("[join]...`

Reply:
thx